css-ecommerce-page/styles.css

```diff
@@ -0,0 +1,42 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "DM Sans", sans-serif;
+  font-size: 0.8rem;
+}
+
+h1 {
+  text-align: center;
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 5%;
+}
+
+img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+.image {
+  height: 300px;
+  width: 200px;
+}
+
+.item {
+  flex-basis: auto;
```

Comment:
Since `flex-basis: auto` is the default value for flex items, you can remove that declaration from the .item styles to simplify your code.